README.md

```diff
@@ -22,7 +22,8 @@ a = b;
 
 After this code executes, what are the values of a and b? Please explain your answer.
 
-<!-- Replace this comment with your answer -->
+Var a = 3
```

Comment:
Kind of a weird way to write this, but yes - those are the right values of `a` and `b`.